webroot/src/models/FormInput/FormInput.model.ts

```diff
@@ -41,6 +41,7 @@ export interface InterfaceFormInput {
     errorMessage?: string | ComputedRef<string>;
     isValid?: boolean;
     isSubmitInput?: boolean;
+    shouldHideErrorMessage?: boolean;
```

Comment:
I think this option may be good to have generally. It seems like it could come up in a few different scenarios. How do you feel about that?

Reply:
I haven't come across a case yet where it's been necessary. It's only a small-ish sample size so far, but I'm also having trouble picturing a need for it. Can you describe the need a bit more?

I was thinking it's easy enough to just unset the error manually for edge case, or just not use the `validation` prop on the form input (it's optional).

Reply:
So I added this specifically for the small expiration date fields. Essentially, it is for cases where for whatever reason we want validation to occur and for the field to be highlighted red when we have validation issues but not display a message.

For the expiration date fields this is because they are tiny and close to each other that finagling it so that the two messages can be displayed and displayed simultaneously in a presentable way didn't seem worth it or as attractive as just highlighting the field red if they leave it blank or dont put it enough characters. 

The idea was just a very easy way to replicate this behavior if the conditions are met where:
1) We want validation
2) The validation issues are sufficiently obvious / the user's input is altered enough to make it so or we want one validation error message for greater than one field
3) Validation error messages would be less attractive than not having validation error messages on selected fields

Reply:
Ok added